File: M8 - Programació multimedia/UF1 Desenvolupament d'aplicacions per a dispositius mòbils/Android Studio/Moreno_Aniol_Activitat1/Prova1/app/src/main/java/com/example/prova1/MainActivity.java
package com.example.prova1;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    // Generals
    private TextView gestio_alumnes;

    private TextView nom_titol;
    private TextView cognoms_titol;
    private TextView edat_titol;
    private TextView poblacio_titol;

    private TextView nom;
    private TextView cognoms;
    private TextView edat;
    private TextView poblacio;



    // Alumne 1
    private TextView alumneView1;
    private Button alumne1;
    private ImageView alumneImage1;

    // Alumne 2
    private TextView alumneView2;
    private Button alumne2;
    private ImageView alumneImage2;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        // General
        gestio_alumnes = findViewById(R.id.textView2);

        nom_titol = findViewById(R.id.textView3);
        cognoms_titol = findViewById(R.id.textView7);
        edat_titol = findViewById(R.id.textView9);
        poblacio_titol = findViewById(R.id.textView11);

        nom = findViewById(R.id.textView4);
        cognoms = findViewById(R.id.textView8);
        edat = findViewById(R.id.textView10);
        poblacio = findViewById(R.id.textView14);


        // Alumne 1
        alumneView1 = findViewById(R.id.textView2);
        alumne1 = findViewById(R.id.button);
        alumneImage1 = findViewById(R.id.imageView);


        alumne1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String nomAlumne = getString(R.string.nom_alumne1);
                String cognomsAlumne = getString(R.string.cognoms_alumne1);
                String gestioAlumnes = nomAlumne + " " + cognomsAlumne;

                gestio_alumnes.setText(gestioAlumnes);

                alumneImage1.setImageResource(R.drawable.foto1);

                nom_titol.setText(getString(R.string.nom_titol));
                cognoms_titol.setText(getString(R.string.cognoms_titol));
                edat_titol.setText(getString(R.string.edat_titol));
                poblacio_titol.setText(getString(R.string.poblacio_titol));

                nom.setText(getString(R.string.nom_alumne1));
                cognoms.setText(getString(R.string.cognoms_alumne1));
                edat.setText(getString(R.string.edat_alumne1));
                poblacio.setText(getString(R.string.poblacio_alumne1));
            }
        });


        // Alumne 2
        alumneView2 = findViewById(R.id.textView2);
        alumne2 = findViewById(R.id.button2);
        alumneImage2 = findViewById(R.id.imageView);

        alumne2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                String nomAlumne = getString(R.string.nom_alumne2);
                String cognomsAlumne = getString(R.string.cognoms_alumne2);
                String gestioAlumnes = nomAlumne + " " + cognomsAlumne;

                gestio_alumnes.setText(gestioAlumnes);

                alumneImage2.setImageResource(R.drawable.foto2);

                nom_titol.setText(getString(R.string.nom_titol));
                cognoms_titol.setText(getString(R.string.cognoms_titol));
                edat_titol.setText(getString(R.string.edat_titol));
                poblacio_titol.setText(getString(R.string.poblacio_titol));

                nom.setText(getString(R.string.nom_alumne2));
                cognoms.setText(getString(R.string.cognoms_alumne2));
                edat.setText(getString(R.string.edat_alumne2));
                poblacio.setText(getString(R.string.poblacio_alumne2));
            }
        });

    }
}

File: M8 - Programació multimedia/UF2 Programació multimèdia/Android studio/exemple_scroll/app/src/main/java/dam2/amoreno/example_scroll/MainActivity.kt
package dam2.amoreno.example_scroll

import android.os.Bundle
import android.widget.TextView
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val pelicules = findViewById<TextView>(R.id.pelicules)
        val series = findViewById<TextView>(R.id.series)
        val esports = findViewById<TextView>(R.id.esports)
        val directes = findViewById<TextView>(R.id.directes)
        val documentals = findViewById<TextView>(R.id.documentals)
        val anime = findViewById<TextView>(R.id.anime)


        pelicules.setOnClickListener{
            Toast.makeText(this, pelicules.text.toString(), Toast.LENGTH_SHORT).show()
        }

        series.setOnClickListener{
            Toast.makeText(this, series.text.toString(), Toast.LENGTH_SHORT).show()
        }

        esports.setOnClickListener{
            Toast.makeText(this, esports.text.toString(), Toast.LENGTH_SHORT).show()
        }

        directes.setOnClickListener{
            Toast.makeText(this, directes.text.toString(), Toast.LENGTH_SHORT).show()
        }

        documentals.setOnClickListener{
            Toast.makeText(this, documentals.text.toString(), Toast.LENGTH_SHORT).show()
        }

        anime.setOnClickListener{
            Toast.makeText(this, anime.text.toString(), Toast.LENGTH_SHORT).show()
        }



    }
}